arch/arm/core/CMakeLists.txt

```diff
@@ -33,3 +33,4 @@ add_subdirectory_ifdef(CONFIG_CPU_CORTEX_M cortex_m)
 add_subdirectory_ifdef(CONFIG_ARM_MPU cortex_m/mpu)
 add_subdirectory_ifdef(CONFIG_CPU_CORTEX_M_HAS_CMSE cortex_m/cmse)
 add_subdirectory_ifdef(CONFIG_ARM_SECURE_FIRMWARE cortex_m/tz)
+add_subdirectory_ifdef(CONFIG_ARM_NONSECURE_FIRMWARE cortex_m/tz)
```

Comment:
This will cause -mcmse to be added to non-secure FW, is this desirable?

Reply:
No, this is not desirable, @SebastianBoe, this should only be needed if we build a _SECURE firmware, thanks for catching this.
We need to avoid that @oyvindronningstad

Reply:
Ok, thanks, added an ifdef.